deploy/examples/cert-manager/istio-csr-helm-values.yaml

```diff
@@ -32,6 +32,6 @@ app:
       port: 6443
 
   controller:
-    rootCAConfigMapName: istio-ca-root-cert
+    rootCAConfigMapName: istio-ca-secret
```

Comment:
`rootCAConfigMapName` is not used in istio-csr templates at all. I already submitted a corresponding [PR](https://github.com/cert-manager/istio-csr/pull/188) to istio-csr.

Reply:
I also remove this from our samples in this [PR](https://github.com/maistra/istio-operator/pull/1101).